pom.xml

```diff
@@ -32,11 +33,16 @@
         <maven-plugin-annotations.version>3.13.0</maven-plugin-annotations.version>
         <eclipse-collections-api.version>11.1.0-r11</eclipse-collections-api.version>
         <maven-plugin-plugin.version>3.13.0</maven-plugin-plugin.version>
+        <maven-clean-plugin.version>3.3.1</maven-clean-plugin.version>
+        <maven-antrun-plugin.version>3.1.0</maven-antrun-plugin.version>
         <jupiter.version>5.12.0</jupiter.version>
         <!-- origin -->
         <origin.working.directory>${project.build.directory}/origin-sources</origin.working.directory>
-        <source.zip>Pilot-Defined-RxNorm-with-SNCT-classes-2024-04-10-with-custom-annotations</source.zip>
+        <source.version>2024-04-10</source.version>
+        <source.zip>Pilot-Defined-RxNorm-with-SNCT-classes-${source.version}-with-custom-annotations</source.zip>
         <origin.namespace>3094dbd1-60cf-44a6-92e3-0bb32ca4d3de</origin.namespace>
```

Comment:
Let's add starterSet and starterSetLocation here as in snomed-ct-data/pom.xml

Reply:
> Let's add starterSet and starterSetLocation here as in snomed-ct-data/pom.xml

Updated - we will now scan the user's downloads directory for the starter ZIP, not the project directory.